app/BlinkLab/BlinkLab.py

```diff
@@ -94,7 +95,14 @@ def printAndLog(msg_type, msg, firstline = False):
                     'msg'      : msg,
                 })+ '\n'
             )
-    
+'''
```

Comment:
please remove this comment. There are many ways to discuss, including GitHub PR reviews and Gist. But not in the source code. The source code is a deliverable, not work in progress.